app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < PostsController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    # @user = User.find(params[:user_id])
+    # @post = Post.find(params[:post_id])
```

Comment:
- Please remove comments from the codebase. They make your code look ugly. Only write necessary comments.